tests/storage/test_user_directory.py

```diff
@@ -72,3 +76,21 @@ def test_search_user_dir_all_users(self):
             )
         finally:
             self.hs.config.user_directory_search_all_users = False
+
+    @defer.inlineCallbacks
+    def test_search_user_dir_stop_words(self):
+        """Tests that a user can look up another user by searching for the start if its
+        display name even if that name happens to be a common English word that would
+        usually be ignored in full text searches.
+        """
+        self.hs.config.user_directory_search_all_users = True
```

Comment:
Thess tests could probably use the `override_config` option instead of setting the config manually this way, but I see that you're just copying from above, so... not huge.

Reply:
ftr I tried using `override_config` but it was't working, I assume that's because the test is a `unittest.TestCase`, not a `unittest.HomeserverTestCase`.

Reply:
Ah, yeah that would do it! 👍